src/main/java/net/cosmocat/marketplace/database/dto/entity/ProductDTO.java

```diff
@@ -0,0 +1,28 @@
+package net.cosmocat.marketplace.database.dto.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import net.cosmocat.marketplace.database.entity.source.AvailabilityStatus;
+
+import java.time.LocalDateTime;
+
+@Data
```

Comment:
why @data? data goest against the principle of the immutability for the dto objects :)